Doc/library/dataclasses.rst

```diff
@@ -173,6 +173,10 @@ Module-level decorators, classes, and functions
      glossary entry for details.  Also see the ``dataclasses.KW_ONLY``
      section.
 
+   - ``slots``: If true (the default is ``False``), :attr:`__slots__` attribute
+     will be generated. If :attr:`__slots__` is defined in the class, then
+     :exc:`TypeError` is raised.
```

Comment:
Could you mention here that if slots is specified then a new class is created and returned? I guess we should also add somewhere that this is the only case where a new class is created, in all other cases the original class is returned by the decorator.